receiver/opencensusreceiver/options_test.go

```diff
@@ -28,7 +30,9 @@ func TestNoopOption(t *testing.T) {
 		opt.withReceiver(subjectReceiver)
 	}
 
-	if !reflect.DeepEqual(plainReceiver, subjectReceiver) {
-		t.Fatalf("noopOption has side effects\nGot:  %+v\nWant: %+v", subjectReceiver, plainReceiver)
-	}
+	require.True(
```

Comment:
nit: this could be `require.Equal` that calls reflect.DeepEqual if necessary and provides a reasonable error messages showing a diff.

Reply:
done